container/Containerfile

```
@@ -14,7 +14,7 @@ ARG CEPH_SHA1
 ARG CEPH_GIT_REPO
 
 # (optional) Define the baseurl= for the ganesha.repo
-ARG GANESHA_REPO_BASEURL="https://buildlogs.centos.org/centos/\$releasever-stream/storage/\$basearch/nfsganesha-5/"
+ARG GANESHA_REPO_BASEURL="https://mirror.stream.centos.org/SIGs/centos/\$releasever-stream/storage/\$basearch/nfsganesha-4/"
```

https://mirror.stream.centos.org/SIGs/centos/9-stream/storage/x86_64/nfsganesha-4/repodata/repomd.xml is a 404

This works: https://mirror.stream.centos.org/SIGs/9-stream/storage/x86_64/nfsganesha-4/, so I think the diff should be (untested):

```suggestion
ARG GANESHA_REPO_BASEURL="https://mirror.stream.centos.org/SIGs/\$releasever-stream/storage/\$basearch/nfsganesha-4/"